paf-mvp-core-js/src/model/generated-model.ts

```diff
@@ -256,6 +262,16 @@ export interface MessageBase {
   timestamp: Timestamp;
   signature: Signature;
 }
+/**
+ * A list of identifiers and a new (unsigned) value for preferences
+ */
+export interface NewUnsignedPreferences {
+  unsignedPreferences?: {
```

Comment:
IMHO, the semantics and compositions of the Preferences models are confusing. If I translate this line:  "new unsigned preferences object that potentially contains unsigned preferences"... 

Can you describe how this class helps us, please? It will help to find an alternative.

More generally speaking, in this file, we have many entities that express the Preferences concept: Preferences, PreferenceData, NewUnsignedPreferences, IdsAndPreferences. I have the impression that we generate classes based on the JSON-Schema and so the readability of the models is impacted by how the JSON-Schema works.

Can it be simplified with less class and more semantics? I want to be constructive so here are a few examples but I may not have all the corner cases in mind.

Example 1, the simplest
```typescript
interface Preferences {
  version: Version;
  data: {
    use_browsing_for_personalization: boolean;
  };
  /** Has a source if the preferences is signed */
  source?: Source;
  /** Weak but explicit relationship to the identifiers. */
  identifiers: Identifier[];
}
// That's it.
```

Exemple 2, more atomical approach
```
interface UnsignedPreferences {
  version: Version;
  data: {
    use_browsing_for_personalization: boolean;
  };
}

interface Preferences : UnsignedPreference {
  source: Source;
}

interface PrebidData { 
  preferences: Preferences;
  identifiers: Identifier[];
}
```